srcpkgs/chatterino2/template

```diff
@@ -0,0 +1,52 @@
+# Template file for 'chatterino2'
+pkgname=chatterino2
+version=2.0.4
+revision=1
+_signals_commit=3f6645c615ff7bf412c05fe322e589cbdd34ff9b
+_settings_commit=ad31b38866d80a17ced902476ed06da69edce3a0
+_humanize_commit=4e00a03623966723f23ca3034c1ad944009cd7be
+build_style=qmake
+hostmakedepends="qt5-qmake qt5-svg-devel qt5-multimedia-devel
+ libcommuni-devel"
+makedepends="qt5-svg-devel qt5-multimedia-devel
+ boost-devel rapidjson libcommuni-devel websocketpp"
+short_desc="Qt-based twitch chat client"
+maintainer="Franc[e]sco <lolisamurai@tfwno.gf>"
+license="MIT"
+homepage="https://github.com/fourtf/chatterino2"
+distfiles="${homepage}/archive/v${version}.tar.gz
+ https://github.com/pajlada/signals/archive/${_signals_commit}.tar.gz
+ https://github.com/pajlada/settings/archive/${_settings_commit}.tar.gz
+ https://github.com/pajlada/humanize/archive/${_humanize_commit}.tar.gz"
+checksum="dd05748a0d7ec8c4e6d1e10ef0da42682dc97fce8dc52b17a276c0bc097e5adf
+e0fca1d0998fad8dbc213d235dd07b3e343f8ebbd0b0af6d15de1bf45dd56aa4
+addb1ec2e06c40e6b467b4bdcdbca0407098fe36009c62ac7e92f157d47b2cb5
+5a5cd8338fcb0a14a4bbe0840f83505d0444a9000956145e85e5550feeb744b0"
+
+case "${XBPS_TARGET_MACHINE}" in
+	armv[56]*)
+		makedepends+=" libatomic-devel"
+		configure_args+="LIBS+=-latomic"
+		;;
+esac
+
+do_install() {
+	for f in chatterino bin/chatterino; do
+		if [ -f "$f" ]; then
+			vbin "$f"
+		fi
+	done
+	vlicense LICENSE
+}
+
+post_extract() {
+	mv "${XBPS_BUILDDIR}/signals-${_signals_commit}/"* lib/signals/
+	mv "${XBPS_BUILDDIR}/settings-${_settings_commit}/"* lib/settings/
+	mv "${XBPS_BUILDDIR}/humanize-${_humanize_commit}/"* lib/humanize/
+}
+
+pre_configure() {
```

Comment:
the functions are not in chronological order of operations

fetch -> extract -> configure -> build -> install -> pkg